common/search/src/main/scala/weco/api/search/elasticsearch/ElasticsearchService.scala

```diff
@@ -92,27 +103,52 @@ class ElasticsearchService(elasticClient: ElasticClient)(
         .map(_.toEither)
         .map {
           case Right(multiResponse) =>
-            val accumulatedTime =
-              multiResponse.items.zipWithIndex.foldLeft(0L) {
-                (acc, itemWithIndex) =>
-                  val (item, index) = itemWithIndex
-
-                  item.response match {
-                    case Right(itemResponse) =>
-                      transaction.setLabel(
-                        s"elasticTook-${index}",
-                        itemResponse.took
-                      )
-                      acc + itemResponse.took
-                    case Left(_) => acc
+            val foldInitial = (
+              0L,
+              Seq.empty[Long],
+              Seq.empty[ElasticsearchError],
+              Seq.empty[SearchResponse]
+            )
+
+            val (
+              finalTotalTimeTaken,
+              finalTimesTaken,
```

Comment:
I'm not sure I love having the total and the list of times tracked separately – feels like an opportunity for them to get out of sync. Could we just track the list and do a `.sum` on the list at the end?

I also wonder if there's a more time-y type we can use rather than `Long`, but I don't think we get that from elastic4s?

Reply:
> I'm not sure I love having the total and the list of times tracked separately

The individual times and the total times are tracked within the same transaction so will be tied together by that.

<img width="962" alt="Screenshot 2021-08-18 at 09 29 47" src="https://user-images.githubusercontent.com/953792/129865398-ec749000-0531-48d5-bb47-ac7e98112c76.png">

^ This is what a transaction looks like in the APM view of Kibana.

> Could we just track the list and do a .sum on the list at the end?

The summed time is interesting because it gives a view on how resource intensive the `multiSearchRequest` was, however it is not representative of the actual time taken as I believe Elasticsearch can execute the constituent queries in parallel. 

An example of where we may want to view the times for individual queries is to distinguish within a multiple request whether a single query within that group is taking a long time to complete.

Reply:
I should have been clearer – I’m fine with them being two individual metrics in Elasticsearch, it just seems odd that we’re managing them as two separate variables in this file.

Reply:
Ah sorry. The variable setup is to avoid side-effecting in a fold.